core/src/appleMain/kotlin/Peripheral.kt

```diff
@@ -74,15 +74,29 @@ import kotlin.coroutines.cancellation.CancellationException
 public actual fun CoroutineScope.peripheral(
     advertisement: Advertisement,
     builderAction: PeripheralBuilderAction,
+): Peripheral = peripheral(advertisement.cbPeripheral, builderAction)
+
+public fun CoroutineScope.peripheral(
+    identifier: Identifier,
+    builderAction: PeripheralBuilderAction,
```

Comment:
Just noticed that this should have a default parameter to match Android's method signature.

```suggestion
    builderAction: PeripheralBuilderAction = {},
```